plugins/versionpress/src/Initialization/.gitignore.tpl

```diff
@@ -8,19 +8,29 @@
 versionpress.maintenance
 /.htaccess
 /web.config
+.well-known/
 
 {{wp-content}}/*
 !{{wp-content}}/db.php
 !{{wp-content}}/index.php
 !{{wp-plugins}}/
 {{wp-plugins}}/versionpress/
+!{{wp-content}}/.versionpress/
 !{{wp-content}}/mu-plugins/
 !{{wp-content}}/themes/
 !{{wp-content}}/languages/
 !{{wp-content}}/uploads/
 !{{wp-content}}/vpdb/
 
 
+#------------------------
+#  Cache files
```

Comment:
I am not sure if this should be added into `.gitignore` template or handled by rules for particular plugins. But it is probably quick win for now and helpfull for users of caching plugins. Folder `/uploads/et_temp` is related to Divi builder?